cluster/provision_gateway.sh

```diff
@@ -0,0 +1,675 @@
+#!/bin/bash
+##############################################################
+# opencontrail-gateway setup and provisioning script. 
+# For more info, please refer to the following link
+# https://github.com/Juniper/contrail-kubernetes
+#
+# Author - Sanju Abraham -@asanju- OpenContrail-Kubernetes
+#
+##############################################################
+set -x
```

Comment:
For all shell scripts, we must do "set -e" and resolve all errors. If an error is ignorable, we can make it a warning (set +e, cmd, set -e) or use || true to ignore the error code.

Otherwise, when errors happen, it will be hard to find and debug them